packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -276,7 +276,8 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         if (!this.channelDeferred) {
             this.channelDeferred = new Deferred<IFluidDataStoreChannel>();
             this.realizeCore(this.existing).catch((error) => {
-                this.channelDeferred?.reject(CreateProcessingError(error, undefined /* message */));
+                this.channelDeferred?.reject(
+                    CreateProcessingError(error, "FluidDataStoreContext.realize", undefined /* message */));
```

Comment:
>FluidDataStoreContext [](http://example.com/codeflow?start=50&length=21)

you did camelCase is the other file, and this is pascal + method. i think it's important to have consistency in our error code or they will just end up as another message which is a mess. my vote is pascal or camel case

Reply:
i consider this blocking. i'd also like to see a comment added whereever we take errorCode that specficies this decision.